### DISCUSSION TITLE
PopOS 20.10 on ASUS ROG Zephyrus G14 Laptop

### DISCUSSION THREAD

**Discussion by julianher** [abskmj/notes#12] [General]
Hey! your guide to install linux on a G14 is awesome. I only faced one issue. Everything in my screen is too small and when I try what you suggest everything gets messed up. Like if I had selected 200% but I selected 125% as you did.

**Comment by abskmj** [comment]
Did you install 20.10 or 21.04? I didn't face any display scalling in the newer version.

**Reply by julianher** [reply]
I installed the 20.04 (LTS) with nvidia drivers. I did not see any 20.10 download option on the official webpage.

**Reply by abskmj** [reply]
Try installing 21.04 version. Some of the things work out of the box.
